certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -76,6 +80,16 @@
                   "type": "boolean",
                   "x-ms-summary": "Mail Enabled"
                 }
+              },
+              "x-ms-test-value": {
+                "description": "A group for development",
+                "displayName": "Test 530930",
+                "groupTypes": [
+                  "Unified"
+                ],
+                "mailEnabled": true,
+                "mailNickname": "Test-Mail-Group-530930",
```

Comment:
We removed this as its bo loner use. Can you please check why this file showing changes other than your change